src/Filesystem/Folder.php

```diff
@@ -355,7 +355,7 @@ public static function isRegisteredStreamWrapper($path)
      * @param string $path Path to check
      * @return string Set of slashes ("\\" or "/")
      *
-     * @deprecated 3.6.5 This method will be removed in 4.0.0. Use correctSlashFor() instead.
+     * @deprecated 3.7 This method will be removed in 4.0.0. Use correctSlashFor() instead.
```

Comment:
Since this patch is targeted to `master` I will go into next `3.6` release hence the version this method will be deprecated in will be `3.6.5`.

Reply:
I changed that because of Markstory put the 3.7 milestone yesterday.
Do you ask to change that or is it not correlated ?